manifest-list.md

```diff
@@ -64,6 +66,13 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
+- **`elements`** *array of objects*
+
+  This OPTIONAL property contains a list of non-manifest objects.
```

Comment:
> there is still the `manifests` field. Why would manifest and manifest-lists be put into this `elements` array?

Maybe I'm not understanding right.  I'd assumed that `elements` would be used when folks were looking for a single-file `refs/` alternative and that `manifests` would only be used for traditional manifest-list stuff (e.g. selecting the best manifest based on platform matching) during some hopefully-short deprecation phase.  In that case, the index would have an `elements` entry pointing at a manifest whenever you create a named ref for a manifest.  And eventually, folks would only use `elements` and we could drop `manifests`.

But it sounds like you are intending all descriptors pointing at `application/vnd.oci.image.manifest.v1+json` go into `manifests` and everything else would go into `elements`?  That works, I guess, although I'm not sure how you'd identify non-OCI types which count as manifests (e.g. a third-party `application/vnd.foo.manifest`).